File: subroutines/selected/subroutine checkforhelix.f
      subroutine checkforhelix(nhxres,dssplab,indexn,iw,hxoklab,ihxok,
     -  lab,llab,maxrsd,maxrec)
      dimension indexn(maxrec)
      character*1 dssplab(maxrsd)
      character*6 hxoklab(3)
      character*(*) lab
      ihxok=1
      do ir=1,nhxres
        indexn(ir)=1
        if (dssplab(ir) .ne. 'G' .and. dssplab(ir) .ne. 'H'
     -    .and. dssplab(ir) .ne. 'I' .and.
     -     dssplab(ir) .ne. 'L') indexn(ir)=0
        if (dssplab(ir) .eq. ' ') dssplab(ir)='?'
      end do
      if (indexn(1) .eq. 0 .or. indexn(nhxres) .eq. 0) then
        ihxok=3
      else
        do ir=2,nhxres-1
          if (indexn(ir) .eq. 0) ihxok=2
        end do
      end if
      write (iw,2088) lab(1:llab),hxoklab(ihxok),
     -  (dssplab(ir),ir=1,nhxres)
      return
2088  format(a,' is ',a,': ',(60a1))
      end

File: fortran to python conversion/todo/to rewrite/subroutine checkforhelix.py
def checkforhelix(nhxres, dssplab, indexn, iw, hxoklab, ihxok, lab, llab, maxresd, maxrec):
	#Probably a random subroutine to retintegrate into analyze
	"""
	Check if helix exits
	***
	Parameters:
	nhxres:
	dssplab: label
	indxn:
	iw:file
	hxoklab: lab
	ihxok:
	lab: label
	llab:
	maxresd:
	maxrec: 
	"""

	ihxok = 1
	for ir in range(nhxres)
		indexn[ir]=1
		if (dssplab[ir] != 'G' and dssplab[ir] != 'H' and dssplab[ir] != 'I' and dssplab[ir] != 'L'):
			indexn[ir] = 0
		if (dssplab[ir] == ' '):
			dssplab[ir] '?'
	if(indxn[0] == 0 or indexn[nhxres] == 0)!
		ihxok = 2
	else:
		for ir in range(1,nhxres-1):
			if (indexn[ir] == 0):
				ihxok = 1
	
	with open(iw, "w+") as file:			
		file.write("{} is {} : {}".format(lab[:llab],hxoklab[ihxok],[dssplab[ir] for ir in range(nhxres) ]))


